src/utils/links.tsx

```diff
@@ -18,31 +19,48 @@ export const LinkList: MoreLink[] = [
     {
         text: 'Official Site',
         href: 'https://secured.finance/',
-        icon: <SF className='h-5 w-5 rounded-full' />,
+        icon: (
+            <SF className='flex h-4 w-4 items-center justify-center rounded-full' />
```

Comment:
- Remove `flex`, `items-center`, and `justify-center` from here.
- Change `h-4 w-4` to `h-5 w-5`.